crates/project_panel/src/project_panel.rs

```diff
@@ -2845,6 +2847,84 @@ mod tests {
         );
     }
 
+    #[gpui::test(iterations = 10)]
```

Comment:
```suggestion
    #[gpui::test]
```

`iterations` are used to run the same test multiple times with different deterministic executor's seed, fuzzing the bugs related to e.g. races around future/task spawns.

We can run this test just once as it seems from its contents — there's no concurrency or hard logic in play?

Reply:
I didn't understood it at first. So just copied this test which looked similar

https://github.com/80avin/zed/blob/d0a971986925157f284507190ed20a1d73ffaf7e/crates/project_panel/src/project_panel.rs#L2717